composer.json

```diff
@@ -25,11 +25,9 @@
 	"require": {
 		"php": ">=5.3.0",
 		"composer/installers": "1.*,>=1.0.1",
-		"mediawiki/semantic-media-wiki": "~1.9|~2.0",
 		"mediawiki/semantic-forms": "~2.8|~3.0"
 	},
 	"require-dev": {
-		"mediawiki/semantic-media-wiki": "@dev"
```

Comment:
This makes me think there is some dependency on SMW at present, either in the production code or in the tests. Need to verify this is not the case.